### PR TITLE
feat(seeding): Ensure admin user while seeding

### PR DESCRIPTION
**Pull request by tomaszdurka**
Create admin user with generated password if not exists
Alternatively if OVERRIDE_ADMIN_PW passed set the password to the value

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@elias-summermatter the main change is that it will always change the password to `OVERRIDE_ADMIN_PW` - not only when admin user does not exist.

File: libs/api/prisma/run-seed.ts [hunk 1]
```diff
@@ -27,47 +28,52 @@ export async function runSeed() {
...
     }
-  })
-  if (!admin) {
-    const email = 'admin@wepublish.ch'
-    const overriddenPw = process.env.OVERRIDE_ADMIN_PW
-    let password = generateSecureRandomPassword(20)
-    if (overriddenPw) {
-      console.log(
-        '\x1b[31m\x1b[1m%s\x1b[0m',
-        `!!!!!!!!!!!!!!!!!! WARNING: UNSECURE PASSWORD IS OVERRIDDEN WITH ENV VARIABLE !!!!!!!!!!!!!!!!!! `
```
Comment (elias-summermatter) [inline libs/api/prisma/run-seed.ts:42]: Maybe keep the warning? What do you think?

File: libs/api/prisma/run-seed.ts [hunk 2]
```diff
@@ -27,47 +28,52 @@ export async function runSeed() {
...
-      `Bootstrapping initial admin user ${email} with password: ${password}`
-    )
+    console.log('\x1b[31m\x1b[1m%s\x1b[0m', `Ensuring admin user`)
     await prisma.user.upsert({
-      where: {
-        id: 'admin'
+      where: {id},
+      update: {
+        ...userBase,
+        password: await bcrypt.hash(password, 11)
```
Comment (elias-summermatter) [inline libs/api/prisma/run-seed.ts:44]: Store password hash in a variable and use hash function just once.  And maybe increase the weight from 11 to 12

**Review by elias-summermatter — CHANGES_REQUESTED** [review]
Besides, this is looks nice, thanks  :+1:

File: libs/api/prisma/run-seed.ts (hunk 2 above)
Comment (tomaszdurka) [reply libs/api/prisma/run-seed.ts:44]: OK

File: libs/api/prisma/run-seed.ts (hunk 1 above)
Comment (elias-summermatter) [reply libs/api/prisma/run-seed.ts:42]: That does not really matter you can change the text, but it's important to notice that it's an unsecure setup.

**Pull request merged**